Pop-Grupa-A-Backend/migrations/versions/8fa29ae8d4f1_user_id_in_cluster_node.py

```diff
@@ -11,7 +11,7 @@
 
 # revision identifiers, used by Alembic.
 revision = '8fa29ae8d4f1'
-down_revision = 'c13c78ae550e'
```

Comment:
Tutaj musi zostać `down_revision = 'c13c78ae550e'` bo inaczej spójności migracji nie ma i sypie błędami. Przynajmniej u mnie

Reply:
<3 bo dropnąłem bazę i nie mogę w żaden sposób migracji zrobić

Reply:
Ale niestety to nie pomogło, coś jeszcze powinno być

Reply:
mój błąd to w migraci add_session_table trzeba down_revision ustawić na  `down_revision = 'c13c78ae550e'`